test/integration_test/snapshot_test.go

```diff
@@ -222,3 +174,73 @@ func parseDataVolumes(
 
 	return dataVolumesNames, dataVolumesInUse
 }
+
+func createSnapshot(t *testing.T) []*scheduler.Context {
+	ctx, err := schedulerDriver.Schedule(generateInstanceID(t, ""),
```

Comment:
If the `testName` arg to `generateInstanceID` is empty, the generated instance ID will be of format "-<uid>". Let's pass the testname here. Will be also useful in debugging failures since the namespace names will have the test name.

Reply:
The namespace is of the format `mysql-snap-group--88602b0b-ae0c-4d4a-8092-7b1350b28ec1`, with the test name in there it hits the length limit usually that's why most of them don't pass in the test name